### ISSUE TITLE
SuiteP - Instalation wizard of SuiteCRM has old style

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2327]
#### Issue

If you instal the fresh instalation of SuiteCRM you will see only the SuiteR style, but default style is SuiteP

**Comment by shogunpol** [comment]
@Mausino , to keeping the installation in his recent appearance is intentional, it is slightly changed, but we try get compromise between improvement and intuitive as possible character of this wizard. However we take seriously your suggestion and consider to improve even more installation wizard.
Thanks for your suggestion.

**Comment by shogunpol** [comment]
@Mausino  thanks for raise this issue, we plan to update the installation wizard, thanks for suggestion.

**Comment by Mausino** [comment]
@shogunpol i think that this was done? I am right?

**Comment by shogunpol** [comment]
@Mausino , yes i thing is done already, if you confirm this is happen please close an issue.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [SuiteP - Instalation wizard of SuiteCRM has old style](https://trello.com/c/98hKfsVL/70-suitep-instalation-wizard-of-suitecrm-has-old-style)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
